modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -492,6 +492,68 @@ def map_func(df, *others):
             ]
         )
 
+    @classmethod
+    @wait_computations_if_benchmark_mode
+    def broadcast_axis(
```

Comment:
Wouldnt it be possible to  reuse `broadcast_axis_partitions` itself if we just add a additional parameter eg `broadcast_right` which defaults to `false` and sends` right_partitions[i] ` to apply if  this is true.